CHANGELOG.md

```diff
@@ -40,6 +40,7 @@ Changelog
 
 ### Chores
 
+- Added tests for currency display settings and stake-pool settings screens ([PR 2622](https://github.com/input-output-hk/daedalus/pull/2622/))
```

Comment:
@ManusMcCole please move this line under "vNext > Chores" and remove the trailing "/" from the URL